feat: add firstinstance handling (#49)
zitadel/zitadel-charts: charts/zitadel/templates/NOTES.txt

Thank you for installing {{ .Chart.Name }}.

> {{- if and (not (empty .Values.zitadel.configmapConfig.FirstInstance)) (and (not (empty .Values.zitadel.configmapConfig.FirstInstance.Org)) (not (empty .Values.zitadel.configmapConfig.FirstInstance.Org.Machine))) }}

Reviewer: does the following not work?
{{- if and .Values.zitadel.configmapConfig.FirstInstance .Values.zitadel.configmapConfig.FirstInstance.Org .Values.zitadel.configmapConfig.FirstInstance.Org.Machine }}

Reviewer: ?

Author: changed it in all ifs

Reviewer: doesn't look changed to me :joy: 
![image](https://user-images.githubusercontent.com/12727842/208081608-3ff1f4dc-11d5-4d79-aff6-debd14b5c068.png)

Author: ah you're right, i just thought that the second 'and' was to much, but the 'not' and 'empty' can be removed too

> If this is you first time deploying ZITADEL, you can now extract your service account key with:

Reviewer: The secret should still be there after `helm upgrade`, right?
```suggestion
Extract your service account key for authenticating at the ZITADEL API's:
```
